src/Umbraco.Core/EmbeddedResources/Lang/en_us.xml

```diff
@@ -410,7 +410,7 @@
     </key>
     <key alias="elementType">Element Type</key>
     <key alias="elementTypeDescription">Defines the schema for a repeating set of properties, for example, in a 'Block
-      List' or 'Nested Content' property editor.
+      List' or 'Block List' property editor.
```

Comment:
Should be "`'Block List` or 'Block Grid' property editor"

Reply:
Good catch! I've fixed it now.